docs/index.md

```diff
@@ -2,33 +2,44 @@
 
 # What is Brainlife?
 
-Brainlife promotes engagement and education in reproducible neuroscience. We do this by providing an online platform where users can publish code (Apps), Data, and make it "alive" by integragrate various HPC and cloud computing resources to run those Apps. 
+brainlife.io promotes engagement and education in reproducible neuroscience by providing an online, community-based platform where users can publish code (Apps) and Data while integrating HPC and cloud-computing resources to run Apps. brainlife.io also allows users to publish all of the research assets associated with their projects that are embedded in our cloud-computing environment and referenced by a single digital-object identifier (DOI). Our platform is unique because of its focus on supporting scientific reproducibility beyond open code and open data -- brainlife.io also provides fundamental smart mechanisms for what we call “Open Services.”
 
-Brainlife also provide mechanisms to publish all research assets associated with a scientific project (data and analyses) embedded in a cloud computing environment and referenced by a single digital-object-identifier (DOI). The platform is unique because of its focus on supporting scientific reproducibility beyond open code and open data, by providing fundamental smart mechanisms for what we refer to as “Open Services.” 
+Below, you’ll learn about the main panel of tools you’ll encounter as soon as you log into brainlife.io. While these run-throughs are brief, you’ll learn much more about each feature as you review the rest of brainlife’s documentation.
```

Comment:
@shmcgavin going over the
```suggestion
Below, you will learn about the main panel of tools you will encounter as soon as you log into brainlife.io. While these run-throughs are brief, you’ll learn much more about each feature as you review the rest of brainlife.io’s documentation.
```

Reply:
Hi @shmcgavin thanks for this! I would prefer if we could avoid contracted forms (we'll or you'll etc.)

Reply:
@francopestilli sorry,  I wasn't sure if I was supposed to resolve these edits but sounds good! I'll making sure not to use contractions as I keep working through documentation.